easybuild/easyblocks/n/nwchem.py

```diff
@@ -43,6 +43,7 @@
 from easybuild.tools.filetools import adjust_permissions, mkdir, write_file
 from easybuild.tools.modules import get_software_libdir, get_software_root, get_software_version
 from easybuild.tools.run import run_cmd
+from easybuild.tools.systemtools import get_avail_core_count, get_total_memory
```

Comment:
@valtandor: this depends on https://github.com/hpcugent/easybuild-framework/pull/1623, please make that clear in the PR description

Reply:
Since easybuilders/easybuild-framework#1623 was merged in log ago, I trust that is no longer necessary.